airbyte-webapp/src/components/Spinner/Spinner.tsx

```diff
@@ -1,43 +1,14 @@
+import classnames from "classnames";
 import React from "react";
-import styled, { keyframes } from "styled-components";
 
-interface IProps {
-  backgroundColor?: string;
+import styles from "./Spinner.module.scss";
+
+interface SpinnerProps {
   small?: boolean;
 }
 
-export const SpinAnimation = keyframes`
-  0% {
-    transform: rotate(0deg);
-  }
-  100% {
-    transform: rotate(360deg);
-  }
-`;
-
-const SpinnerWheel = styled.div<{ small?: boolean }>`
-  display: inline-block;
-  height: ${({ small }) => (small ? 30 : 42)}px;
-  width: ${({ small }) => (small ? 30 : 42)}px;
-  border-radius: 50%;
-  border: 4px solid ${({ theme }) => theme.primaryColor12};
-  position: relative;
-  animation: ${SpinAnimation} 1.5s linear 0s infinite;
-`;
-
-const BreakRec = styled.div<IProps>`
-  width: 13px;
-  height: 7px;
-  background: ${({ theme, backgroundColor }) => (backgroundColor ? backgroundColor : theme.whiteColor)};
```

Comment:
Did we need to preserve backgroundColor?

Reply:
No actually - that was a hack, to make a "break" in the circle to match the background color.
<img width="138" alt="image" src="https://user-images.githubusercontent.com/20929344/185180401-a5e91194-1a29-4d3f-9506-dfd9b5240ed2.png">